area/sin.py

```diff
@@ -133,12 +133,12 @@ def facing_sin():
 
 def inside_sin():
     logger.info("Moving to position next to save sphere")
-    while not pathing.set_movement([247, -237]):
-        if memory.main.diag_skip_possible():
-            xbox.tap_b()
-        elif memory.main.menu_open():
-            xbox.tap_b()
-    logger.info("Moving to next map")
+    #while not pathing.set_movement([247, -237]):
```

Comment:
We sure we don't need this? Is something else doing this now?

Reply:
Tested and validated before the PR was put up. This was caused due to over-testing any% CSR with under-testing other categories that lead into this point of the game.